internal/client/api_version.go

```diff
@@ -160,6 +171,11 @@ func EncodeRequest(req *tikvrpc.Request) (*tikvrpc.Request, error) {
 		r := *req.RawCompareAndSwap()
 		r.Key = EncodeV2Key(ModeRaw, r.Key)
 		newReq.Req = &r
+	case tikvrpc.CmdRawChecksum:
+		r := *req.RawChecksum()
+		// ranges in req maybe reused when retry, alloc another slice to avoid changing the range.
```

Comment:
Suggest to move the comment on top of this function, to give a warning when others add a new interface.

And suggest words like:
NOTE: `req` is reused on retry. MUST encode on cloned request, other than overwrite the original.